tests/test_other.py

```diff
@@ -9069,3 +9069,26 @@ def test_link(self):
     rval = send (0, thing, 0, 0);
     return 0;
 }''', '', force_c=True)
+
+  # This test verifies that function names embedded into the build with --js-library (JS functions imported to asm.js/wasm)
+  # are minified when -O3 is used
+  def test_js_function_imports_are_minified(self):
+    def check_size(f, expected_size):
+      if not os.path.isfile(f):
+        return # Nonexistent file passes in this check
+      obtained_size = os.path.getsize(f)
+      print('size of generated ' + f + ': ' + str(obtained_size))
+      try_delete(f)
+      assert obtained_size < expected_size
+
+    run_process([PYTHON, path_from_root('tests', 'gen_many_js_functions.py'), 'library_long.js', 'main_long.c'])
+    for wasm in [['-s', 'WASM=0', '--separate-asm']]: # TODO: Add support to Wasm to minify imports, and then add Wasm testing ['-s', 'WASM=1'] to this list
+      for closure in [['--closure', '1']]: # Currently we rely on Closure for good minification of the JS function names. TODO: Add minification also for non-Closure users and add [] to this list to test minification without Closure.
```

Comment:
Please put comments on the preceding line and wrap at 80.

Reply:
Previously tailing comments have been fine, can flake warn about these if that is now a mandated style thing?

Reply:
Its not strictly part of style guide I guess, but in cases like this where it makes the line longer than 80 I think it makes the code hard to read.

Ideally we'd be able to enforce some kind of line length limit, but its hard because (8) flake8 is not configurable in this way (b) there is so much code with long lines in emscripten already.

There was a recent discussion about limiting line length to 80 for new code, but we didn't decide to do that yet, and it would take a little effort to enforce programatically.

Anyway, its not a big deal here.